### PR TITLE
Using github latest release url to check for software update

### PR DESCRIPTION
**Pull request by beelchester**
https://github.com/rustdesk/rustdesk/discussions/5475#discussioncomment-6790057

https://github.com/rustdesk/rustdesk/assets/73148455/3b85268b-afe5-42fe-9f4b-12810d4c66d3

### CODE REVIEW

File: src/common.rs [hunk 1]
```diff
@@ -831,30 +831,19 @@ pub fn check_software_update() {
...
-            }
-        }
+    let url = "https://github.com/rustdesk/rustdesk/releases/latest";
+    let latest_release_response = reqwest::get(url).await?;
+    let latest_release_version = latest_release_response
+        .url()
+        .path()
+        .rsplit('/')
+        .next()
+        .unwrap();
```
Comment (rustdesk) [inline src/common.rs:841]: potential crash here (unwrap)

Comment (rustdesk) [reply src/common.rs:841]: Never use unwrap except have to use, e.g. mutex.

Comment (beelchester) [reply src/common.rs:841]: Ok, got it.
Looks like you have fixed it https://github.com/rustdesk/rustdesk/commit/28ddf6cf079a5cad14e75b1e311596c026fef326 
Thanks.

Comment (rustdesk) [reply src/common.rs:841]: one-day effort we used to position this crash since it happened in very specific case only.

**Pull request merged**